derp/derp_server.go

```diff
@@ -1955,6 +1955,22 @@ func (s *Server) ConsistencyCheck() error {
 			s.curClients.Value(),
 			len(s.clients)))
 	}
+	if s.verifyClientsLocalTailscaled {
```

Comment:
can you write a brief comment here to help readers understand what this is doing?